gradle/libs.versions.toml

```diff
@@ -35,7 +35,7 @@ nanohttpd =  "2.3.1"
 napier = "2.7.1"
 mockwebserver =  "4.12.0"
 mockito-kotlin =  "6.0.0"
-
+credentials =  "1.3.0-rc01"
```

Comment:
should be androidx-credentials as per the convention used in the file - same applies to module names (not credentials-androidx)

Reply:
Not yet resolved: library name needs to match the pattern.